.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+    "env": {
+        "es6": true
+    }
+}
```

Comment:
This is duplicate with `.eslintrc`, please move it there or move all options to this file. I prefer to use filename `.eslintrc` because its the most compatible (other file extensions came later).

Reply:
Actually, we already have `env: es6` there, so this file should be completely useless as it is.